client/my-sites/people/invite-status/index.jsx

```diff
@@ -9,16 +9,11 @@ const InviteStatus = ( {
 	handleDelete,
 	resendSuccess,
 	requestingResend,
-	inviteWasDeleted,
 	deletingInvite,
 } ) => {
 	const translate = useTranslate();
 	const { isPending } = invite;
 
-	if ( invite && inviteWasDeleted ) {
```

Comment:
Hi @agrullon95 To keep the pending list up to date, I had to remove this logic added by you in https://github.com/Automattic/wp-calypso/pull/86927. Could you please help me to test and make sure it is still working well?

Reply:
I don't recall why I added that there, but I didn't notice any issues with invitations/revokes when testing.